electron/renderer/src/components/Webview.js

```diff
@@ -58,15 +58,15 @@ class Webview extends Component {
   }
 
   _focusWebview() {
-    const is_visible = this.props.visible === true;
-    if (is_visible) {
+    const isVisible = this.props.visible === true;
+    if (isVisible) {
```

Comment:
```js
if (this.props.visible === true) {
```

Reply:
or just `if (this.props.visible)`?

Reply:
I prefer the explicit test.

Reply:
plus [it's shorter](https://github.com/wireapp/wire-desktop/pull/1382#discussion_r184029420) :)